faq.html

```diff
@@ -4,4 +4,16 @@
 ---
 
 <h3>Frequently Asked Questions</h3>
-todo
\ No newline at end of file
+1. Why the name "in-toto"?
+in-toto is Latin for “as a whole". It is the first framework that holistically enforces the integrity of a software supply chain by gathering cryptographically verifiable information about the chain itself, from development to installation of the software.
+
+2. What is the difference between in-toto and TUF?
+The Update Framework (TUF) and in-toto are two faces of a coin. While TUF provides a framework (a set of libraries, file formats, and utilities) that can be used to secure new and existing software update systems, in-toto scans the entire software supply chain to make sure that all the stakeholders involved in development, build and shipment of the software produce exactly what they are supposed to. Together, they provide end-to-end protection for the whole software supply chain.
```

Comment:
I think two faces of a coin may be a little confusing for non-native speakers. I think that first sentence can go away, and we could change the second to something like:
```
TUF provides a framework that can be used to secure update systems (i.e., the last mile), whereas in-toto lets you verify the whole software supply chain. TUF and in-toto can play together very well, as you can use TUF to deliver updates and their corresponding in-toto metadata.
```

Reply:
I have made the changes in the answer as per your suggestion.